scss/bootstrap/overrides/crm/_error.scss

```diff
@@ -1,12 +1,9 @@
 .crm-error {
   border-radius: $border-radius-small;
+  display: inline-block;
+  margin-top: $crm-btn-padding-small-vertical;
 }
 
 input[type=file].error {
   padding-left: $crm-btn-padding-small-vertical;
```

Comment:
I know that is not your code, but it just looks wrong. Please see the comment above about `$crm-btn-padding-small-vertical`

Reply:
this is actually mine :P